Cargo.toml

```diff
@@ -9,10 +9,13 @@ authors = ["Gear Technologies"]
 gstd = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
 multitoken-io = { git = "https://github.com/gear-dapps/multitoken.git", tag = "0.3.3" }
 gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.2" }
+hashbrown = { version = "0.13.1" }
 
 [build-dependencies]
 gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
 
 [dev-dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], branch = "stable" }
+gstd = { git = "https://github.com/gear-tech/gear.git", features = [
+    "debug",
+], branch = "stable" }
```

Comment:
```suggestion
gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], branch = "stable" }
```

Reply:
**debug** feature add implementation  `gstd::From<u64>`  for  `gstd::ActorId`

Another way we have compile errors:
`gstd::From<u64>` is not implemented for `gstd::ActorId`

Reply:
I didn't suggest removing this feature. I suggested keeping dependencies in one line for a common style.